src/frontend/OverlayManager/Overlay/index.tsx

```diff
@@ -147,6 +149,7 @@ export const Overlay = observer(function ({
           <div className={BrowserGameStyles.rightSideContainer}>
             {exitGameButton}
             {extensionManager}
+            <QuestRewardClaimedToast />
```

Comment:
this won't show if the overlay is hidden. I think this might overlap the existing transaction notification and initial toasts. Let's remove it and explore adding more variety to the overlay toasts on a separate PR

Reply:
You're right; I think it is fine for now, though. That's the position the design team placed it too: https://www.figma.com/design/GKPELZxk551oVC3EDLgEDA/Overlay?node-id=14764-21449&node-type=frame&m=dev

<img width="1280" alt="Screenshot 2024-11-12 at 3 53 10 PM" src="https://github.com/user-attachments/assets/9a4c7cee-7ef2-4d12-926b-83899aa13686">

Reply:
We decided to remove it and explore in follow up PRs